scipy/stats/_continuous_distns.py

```diff
@@ -5026,7 +5026,50 @@ class levy_gen(rv_continuous):
 
     %(after_notes)s
 
-    %(example)s
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from scipy.stats import levy
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots(1, 1)
+
+    Calculate the first four moments:
+
+    >>> mean, var, skew, kurt = levy.stats(moments='mvsk')
+
+    Display the probability density function (``pdf``):
+
+    >>> x = np.linspace(levy.ppf(0.0),
+    ...                 levy.ppf(0.6), 100)
+    >>> ax.plot(x, levy.pdf(x),
+    ...        'r-', lw=5, alpha=0.6, label='levy pdf')
+
+    Alternatively, the distribution object can be called (as a function)
+    to fix the shape, location and scale parameters. This returns a "frozen"
+    RV object holding the given parameters fixed.
+
+    Freeze the distribution and display the frozen ``pdf``:
+
+    >>> rv = levy()
+    >>> ax.plot(x, rv.pdf(x), 'k-', lw=2, label='frozen pdf')
+
+    Check accuracy of ``cdf`` and ``ppf``:
+
+    >>> vals = levy.ppf([0.001, 0.5, 0.999])
+    >>> np.allclose([0.001, 0.5, 0.999], levy.cdf(vals))
+    True
+
+    Generate random numbers:
+
+    >>> r = levy.rvs(size=1000)
+
+    And compare the histogram:
+
+    >>> r = r[r <= x[-1]]
```

Comment:
Oops. This essentially truncates the distribution. Need to normalize the histogram manually, I guess?

Reply:
This looks strange to me actually. Why doing this at all?

Reply:
The tail is super long and fat. The huge random variates are what gives `hist` auto-binning trouble. Moreover, if you don't zoom in, all you see is a spike. So somehow we want to zoom in on the relevant region and avoid passing all those points to `hist`. There could be other ways, but this is one way.

Reply:
Ah, makes sense! Maybe then give this one a separate doc string example, not the general one? 
(I sort of wonder if e.g. cauchy not have the same problem)

Reply:
Not as bad, apparently.
I plan to fix the normalization and add comments if I have to do anything unusual (like this), then merge. I'm just trying to finish up dormant PRs. But you're welcome to make-better in a follow-up!

Reply:
Sounds good to me!